include/sys/fs/zfs.h

```diff
@@ -977,9 +997,18 @@ typedef enum zpool_errata {
 	ZPOOL_ERRATA_ZOL_6845_ENCRYPTION,
 } zpool_errata_t;
 
+/*
+ * For backwards compatibility with the existing user space utilities the
```

Comment:
can you add here the specific thing that you are trying to keep the same - `vdev_stat` structure?

Reply:
Fixed.